cmake/SPIRV-Tools-shared.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
```

Comment:
Its important to leave the `$[prefix]` alone and append only the relative directory if applicable. It should only be an absolute path if its not actually inside `$[prefix]`.

Reply:
I agree, but I don't know an easy way to achieve that with CMake. If anyone has suggestions, I can amend the PR.

Reply:
Unfortunately I am not sure how in cmake, but its basically a matter of comparing strings and reacting accordingly.

* If `CMAKE_INSTALL_LIBDIR` and  and `CMAKE_INSTALL_INCLUDEDIR` are absolute paths, but within `CMAKE_INSTALL_PREFIX` then strip the prefix and prepend `${prefix}`.
* If they are relative paths, just prepend `${prefix}`.
* If they absolute paths, but not within `CMAKE_INSTALL_PREFIX` then add the full path without prepending `${prefix}`.

Its often easiest to just use relative paths...

Reply:
@akien-mga I spent more time on this and here is a working example.

https://github.com/FNA-XNA/FAudio/commit/cedb8c6d2a07279cdb6a2f960150a47eccf1dcc3

Reply:
@akien-mga: do you have time to circle back to this?